content/assets/css/_typography.scss

```diff
@@ -23,26 +31,58 @@ h1 {
 
 h2 {
   font-size: 1.4em;
-  margin: $base-unit/1.5 0;
+  margin: 25px 0 10px 0;
 }
 
 h3 {
   font-size: 1em;
   font-weight: bold;
-  margin-bottom: $base-unit/1.5;
+  margin: 18px 0 4px 0;
   i {
     margin-right: $base-unit/4;
   }
 }
 
-p {
-  margin: 0 0 $base-unit/1.5 0;
+h5 {
+  font-weight: bold;
+  font-size: 15px;
 }
 
-pre {
+p, pre, table {
   margin: 0 0 $base-unit/1.5 0;
 }
 
 a {
   color: $accent-color;
+  transition: all 0.2s ease;
+}
+
+hr {
+  margin: 20px 0;
+  border: 0;
+  border-top: 1px solid #eee;
+}
+
+code {
+  padding: 2px 4px;
```

Comment:
This is causing a visual bug, where the content is no longer aligned across the body of the code. Please refer to the screenshot.

<img width="731" alt="Screenshot 2021-01-08 at 9 23 31" src="https://user-images.githubusercontent.com/17004703/103986551-a11ce580-5193-11eb-9a15-dbb4ae566ebd.png">

Reply:
Great find! This is fixed!
<img width="636" alt="Screen Shot 2021-01-08 at 8 40 29 AM" src="https://user-images.githubusercontent.com/80610/103987741-42ed0400-518d-11eb-8f1d-9d4d7739efca.png">